### DISCUSSION TITLE
Why do I see two different colors when using {background-color}?

### DISCUSSION THREAD

**Discussion by ghost** [quarto-dev/quarto-cli#7618] [Q&A] [answered]
### Description

Hello there,

When customizing a slide background with 
```qmd
## Slide backgrounds {background-color="#806040"}

Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.
```
it renders a slightly different colored rectangle around the content part. See screenshot:
<img width="1016" alt="screenshot#" src="https://github.com/quarto-dev/quarto-cli/assets/143960140/78690844-a84b-43ba-a8b2-53020ca047cb">

There is nothing fancy in my presentation, that to my knowledge could cause this odd behaviour. The YAML header is quite basic:
 ```qmd
---
title: "My title"
format:
  revealjs:
    incremental: false
    scrollable: true
    theme: dark
editor: visual
mainfont: avenir
---
```

Can anyone help me out with this?
Many thanks in advance!

**Comment by cderv** [comment]
DO you have any other customization ? I can't reproduce with this 

````
---
title: test
format:
 revealjs:
   theme: dark
---

## Slide backgrounds {background-color="#806040"}

Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.

````

![image](https://github.com/quarto-dev/quarto-cli/assets/6791940/5e10bc44-1a28-4787-a708-2b4b0eb17be1)

**Reply by ghost** [reply, marked as answer]
Thanks for your reply. No, I do not have any further customization. However, I finally quit RStudio and re-rendered the presentation. Then it was gone. Still odd, why it occured in the first place.
